src/main/java/io/kestra/plugin/microsoft365/outlook/Send.java

```diff
@@ -0,0 +1,257 @@
+            full = true,
+            code = """
+                id: send_detailed_email
+                namespace: company.team
+
+                tasks:
+                  - id: send_email
+                    type: io.kestra.plugin.microsoft365.outlook.Send
+                    auth:
+                      tenantId: "{{ secret('AZURE_TENANT_ID') }}"
+                      clientId: "{{ secret('AZURE_CLIENT_ID') }}"
+                      clientSecret: "{{ secret('AZURE_CLIENT_SECRET') }}"
+                      userPrincipalName: "sender@example.com"
+                    to:
+                      - "primary@example.com"
+                    cc:
+                      - "cc1@example.com"
+                      - "cc2@example.com"
+                    bcc:
+                      - "bcc@example.com"
+                    subject: "Important Notification"
+                    body: "This is a plain text email sent from Kestra workflow."
+                    bodyType: "TEXT"
+                """
+        )
+    }
+)
+public class Send extends Task implements RunnableTask<Send.Output> {
+
+    @Schema(
+        title = "Authentication configuration",
+        description = "Microsoft Graph authentication settings including tenant ID, client ID, and client secret"
+    )
+    @Valid
+    @NotNull
+    private GraphAuthConfig auth;
+
+    @Schema(
+        title = "To recipients",
+        description = "List of email addresses to send the email to"
+    )
+    @NotNull
+    private Property<List<String>> to;
+
+    @Schema(
+        title = "CC recipients",
+        description = "List of email addresses to carbon copy"
+    )
+    private Property<List<String>> cc;
+
+    @Schema(
+        title = "BCC recipients",
+        description = "List of email addresses to blind carbon copy"
+    )
+    private Property<List<String>> bcc;
+
+    @Schema(
+        title = "Email subject",
+        description = "Subject line of the email"
+    )
+    @NotNull
+    private Property<String> subject;
+
+    @Schema(
+        title = "Email body",
+        description = "Body content of the email"
+    )
+    @NotNull
+    private Property<String> body;
+
+    @Schema(
+        title = "Body type",
+        description = "Content type of the email body (HTML or TEXT)",
+        defaultValue = "HTML"
+    )
+    private Property<String> bodyType;
+
+    @Schema(
+        title = "From address",
+        description = "Email address to send from (optional, uses authenticated user if not specified)"
+    )
+    private Property<String> from;
+
+    @Override
+    public Output run(RunContext runContext) throws Exception {
+        Logger logger = runContext.logger();
+
+        // Create Graph service client
+        GraphServiceClient graphClient = GraphService.createClientCredentialClient(auth, runContext);
+
+        // Render properties
+        List<String> toRecipients = runContext.render(to).asList(String.class);
```

Comment:
Please prefix all rendered properties with "r", it's a convention for us to understand in the code it comes from a rendered property.